app/steps/ui/stoppage/template.html

```diff
@@ -1,21 +1,18 @@
 {% extends "includes/layout.html" %}
 
 {% block page_title %}
-    {{ content.title}}
+    {{content.title}}
```

Comment:
Could leave this out with the new changes merged into Demo that take the title into the inherited "layout.html" file.